package.json

```diff
@@ -20,24 +20,26 @@
   },
   "dependencies": {
     "@bigcommerce/big-design": "^0.34.3",
```

Comment:
since we've just upgraded React, as well as the other BD packages, this might also be a good time to update the main BigDesign package to 0.36

Reply:
Updated to 36